webpack.config.cjs

```diff
@@ -2,6 +2,8 @@ const path = require( 'node:path' );
 const CopyPlugin = require( 'copy-webpack-plugin' );
 const { TsconfigPathsPlugin } = require( 'tsconfig-paths-webpack-plugin' );
 const FileManagerPlugin = require( 'filemanager-webpack-plugin' );
+// eslint-disable-next-line import/no-extraneous-dependencies
+const webpack = require( 'webpack' );
```

Comment:
```suggestion
const ProvidePlugin = require( 'webpack' ).ProvidePlugin;
```

Reply:
@psrpinto Not sure I follow this suggestion?

Reply:
Sorry, I should've also mentioned that if we would do it like my suggestion, than we can do:

```ts
const webExtensionPolyfillPlugin = new ProvidePlugin( {
    browser: 'webextension-polyfill',
} );
```

It would be consistent with how other plugins are referenced e.g. `FileManagerPlugin`.

Just a detail, fee free to ignore.

Reply:
Alternatively, we could do (at the top level, outside any function):

```
const webExtensionPolyfillPlugin = new (require( 'webpack' ).ProvidePlugin)( {
	browser: 'webextension-polyfill',
} );
```

Again, just a suggestion, feel free to ignore.

Reply:
I like your second suggestion, but I think current one is simpler, so I will just keep it as is if you don't mind :)

Reply:
We can change it as webpack config evolves further, whatever makes more sense at that point.